.github/workflows/add-review-team.yml

```diff
@@ -0,0 +1,31 @@
+  name: Label PRs
+
+  on:
+    issue_comment:
+      types: [created]
```

Comment:
Should this be the only `issue_comment` or should we also consider `pull_request_review_comment`?

Also, I restricted this to `created` To make sure that correcting spelling mistakes etc. don't trigger it but does anyone know of a situation where this might be helpful?

Ref:
* https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review_comment
* https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment